boa_engine/src/vm/flowgraph/mod.rs

```diff
@@ -526,9 +526,9 @@ impl CodeBlock {
             }
         }
 
-        for function in self.functions.as_ref() {
-            let subgraph = graph.subgraph(String::new());
-            function.to_graph(interner, subgraph);
-        }
+        // for function in self.functions.as_ref() {
+        //     let subgraph = graph.subgraph(String::new());
+        //     function.to_graph(interner, subgraph);
+        // }
```

Comment:
Missing deletion? If not, it would be good to add a `TODO` explaining why this is commented out for now.